week-1/errors/2.js

```diff
@@ -1,5 +1,12 @@
 // Currently trying to print the string "I was born in Bolton" but it isn't working...
 // what's the error ?
 
-console.log(`I was born in ${cityOfBirth}`);
+// Cannot access 'cityOfBirth' before initialization.
+// The variable should be declared first
+
+// console.log(`I was born in ${cityOfBirth}`);
+// const cityOfBirth = "Bolton";
```

Comment:
Good explanation. Feel free to delete lines that are no longer in use!

Reply:
Thank you, of course!